data-model/fdo-profile/virtual-collection/0.1.0/schema/virtual-collection.json

```diff
@@ -150,15 +150,10 @@
       "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_collectionName A short title that summarizes the collection objects contained within the ObjectGroup",
       "$comment": "idx = 900"
     },
-    "conditionsOfAccess": {
```

Comment:
Believe this is more for physical collections. The first version of the virtual collection will always be public